static/js/gallery.js

```diff
@@ -0,0 +1,26 @@
+function setupGallery(galleryContainer, imgElement, images) {
```

Comment:
I would wrap the JS with an IIFE like this: https://github.com/mixxxdj/website/blob/website/static/js/download.js, exposing a namespace like this: https://github.com/mixxxdj/website/blob/website/static/js/mixxx.js

Reply:
I deliberately decided against an IIFE since we have `let` and `const`now.